packages/@jsii/python-runtime/requirements.txt

```diff
@@ -6,5 +6,6 @@ pytest-mypy~=0.10
 setuptools~=75.3.2
 types-python-dateutil~=2.9
 wheel~=0.45
+orjson>=3.10,<4
```

Comment:
¿Quanto vale?

Reply:
Couldn't get this to work. I was hoping for an easy performance improvement, but it isn't easy.